src/PBot/FodyWeavers.xml

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
For the record, this is the removal of the BOM. Not a problem, since UTF-8 doesn't care whether it's there or not.